app/views/layouts/listnav/_host.html.haml

```diff
@@ -119,6 +119,15 @@
             :record_id     => @record.id,
             :title         => _("Show %{host} drift history") % {:host => host_title})
 
+        - if role_allows?(:feature => "physical_server_show") && !@record.physical_server.nil?
+          %li
+            = link_to("#{ui_lookup(:table => "physical_servers")}: #{@record.physical_server.name}",
```

Comment:
For this `ui_lookup(:table => "physical_servers")` (note the singular use of `:table` as opposed to `:tables`), I was expecting to see a singular `Physical Server`, but instead seeing a plural (`Physical Servers`)
<img width="328" alt="screen shot 2017-06-05 at 11 37 54 am" src="https://cloud.githubusercontent.com/assets/1538216/26797918/7c436d44-49e3-11e7-9cc7-60d2ad5e79a1.png">

@mzazrivec Any idea why?